flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -579,22 +578,18 @@ private Operation convertCreateDatabase(SqlCreateDatabase sqlCreateDatabase) {
         String databaseName =
                 (fullDatabaseName.length == 1) ? fullDatabaseName[0] : fullDatabaseName[1];
         boolean ignoreIfExists = sqlCreateDatabase.isIfNotExists();
-        String databaseComment =
-                sqlCreateDatabase
-                        .getComment()
-                        .map(comment -> comment.getValueAs(NlsString.class).getValue())
-                        .orElse(null);
+        String databaseComment = OperationConverterUtils.getComment(sqlCreateDatabase.getComment());
```

Comment:
following my comment above:
```suggestion
        String databaseComment = sqlCreateDatabase.getComment();
```